common/src/stack/command/stack/commands/load/json/plugin_appliance.py

```diff
@@ -0,0 +1,208 @@
+# @copyright@
+# Copyright (c) 2006 - 2018 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+import stack.commands
+import json
+from stack.exception import CommandError
+
+class Plugin(stack.commands.Plugin, stack.commands.Command):
+	notifications = True
```

Comment:
What's the purpose of this line?  Does this in some way enable self.notify()?  If so, does it work to set it once in load's `init.py`, or does it need to be set for every plugin?  Not necessary to "fix" this, I just don't want to check in redundant LOC.

Reply:
From the examples I was looking at, it is required. An attribute error is raised if it is missing.
AttributeError -- 'Plugin' object has no attribute 'notifications'